core/src/main/kotlin/com/justai/jaicf/builder/Scenario.kt

```diff
@@ -0,0 +1,36 @@
+package com.justai.jaicf.builder
+
+import com.justai.jaicf.api.BotRequest
+import com.justai.jaicf.generic.ChannelTypeToken
+import com.justai.jaicf.model.ScenarioModelBuilder
+import com.justai.jaicf.model.scenario.Scenario
+import com.justai.jaicf.model.scenario.createScenario
+import com.justai.jaicf.reactions.Reactions
+
+@ScenarioDsl
+fun <B : BotRequest, R : Reactions> Scenario(
+    channelToken: ChannelTypeToken<B, R>,
+    body: ScenarioBuilder<B, R>.() -> Unit
+): Scenario = createScenario { ScenarioBuilder(channelToken).apply(body).build() }
+
+@ScenarioDsl
+fun Scenario(
+    body: ScenarioBuilder<BotRequest, Reactions>.() -> Unit
+): Scenario = Scenario(ChannelTypeToken.Default, body)
+
+@ScenarioDsl
+fun <B : BotRequest, R : Reactions> startScenario(
```

Comment:
Why not `createScenario` or `buildScenario`?

Reply:
To indicate that it's related to `start` function

Reply:
Since method `createScenario` became internal, I think we can reconsider `startScenario` naming, and `start` method itself.
Even if we still keep `start` method, I think naming is incorrect. We don't start anything, we don't launch any processing, we're `creating` or `building` a scenario.